### PR TITLE
[ONNX] Update ONNX importer to use RNN/GRU Sequence ops

### PR DESCRIPTION
**Pull request by mitruska**
Enable usage of opset5 RNN/GRU Sequence ops instead of custom importer recurrent implementation. 
Most of the "changed files" are just new onnx model prototxt files.

Logic changes:
- [ngraph/frontend/onnx_import/src/op/gru.cpp ](https://github.com/openvinotoolkit/openvino/pull/2035/files#diff-968757ba3e796c5fa29392d4edfce7cd)

- [ngraph/frontend/onnx_import/src/op/rnn.cpp ](https://github.com/openvinotoolkit/openvino/pull/2035/files#diff-223b51f7bc8aea334a286b5093251786)

- [ngraph/frontend/onnx_import/src/utils/recurrent.cpp](https://github.com/openvinotoolkit/openvino/pull/2035/files#diff-1bc09bf263ea67763ff1dc5aa6eae28e)

Current CPU plugin implementation requires W, R, B inputs as constant.
So all of the onnx model tests are now in two versions with inputs as parameters for example `onnx_model_gru_defaults_fwd`
and as constants with `_const` at the end of name like `onnx_model_gru_defaults_fwd_const`.

New ONNX model tests with const W, R, B input: 
-  [ngraph/test/onnx/onnx_import_rnn.in.cpp](https://github.com/openvinotoolkit/openvino/pull/2035/files#diff-76e54f63e2e1d54310efd77b315fc288)

### CODE REVIEW

**Review by arogowie-intel — APPROVED** [review]
The ONNX Import part looks reasonable.

File: ngraph/frontend/onnx_import/src/utils/recurrent.cpp [hunk 1]
```diff
@@ -37,7 +37,7 @@ namespace ngraph
             {
```
Comment (mbencer) [inline ngraph/frontend/onnx_import/src/utils/recurrent.cpp:37]: As I understand `RecurrentSequence` class can be removed now?

Comment (mitruska) [reply ngraph/frontend/onnx_import/src/utils/recurrent.cpp:37]: It's really nice piece of code, but you're right, we can remove it, in favour of the RNN/GRU Sequence classes

**Review by postrational — APPROVED** [review]

File: ngraph/frontend/onnx_import/src/utils/recurrent.cpp (hunk 1 above)
Comment (mitruska) [reply ngraph/frontend/onnx_import/src/utils/recurrent.cpp:37]: Done

**Review by itikhono — APPROVED** [review]

**Review by mbencer — APPROVED** [review]

**Pull request merged**